extensions/cuid-generator/package.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "cuid-generator",
+  "title": "CUID Generator",
+  "description": "A quick way to generate CUIDs",
+  "icon": "command-icon.png",
+  "author": "dgrcode",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generate",
+      "title": "Generate cuid",
+      "description": "Generates a new cuid",
+      "mode": "view"
```

> One possible bug is at the final state. Raycast closes, but when I open raycast again it is "inside" the cuid generator extension, which doesn't have any interface, so it's sitting on an empty raycast window. Gif below:

The reason for this behaviour is that the command is a `view` command, when it doesn't actually render any view. 
```suggestion
      "mode": "no-view"
```